org.adempiere.base/src/org/compiere/util/CacheMgt.java

```diff
@@ -307,7 +307,7 @@ protected int resetLocalCache (String tableName, int Record_ID)
 		CacheInterface[] instances = getInstancesAsArray();
 		for (CacheInterface stored : instances)
 		{
-			if (stored != null && stored instanceof CCache)
+			if (stored != null && stored.size() > 0)

```

Comment:
To avoid a potential classcast exception below, it is safer to keep the instanceof check as well.

Reply:
done